tests/cypress/e2e/spec.cy.ts

```diff
@@ -54,21 +54,13 @@ describe('e2e tests', () => {
     cy.visit('/dashboard/new');
     cy.get('button[aria-label="Add new panel"]').click();
 
-    cy.get('input[id="react-select-7-input"]').type('Host name{enter}'); // Filter -> 'Host name'
+    cy.get('input[id="react-select-7-input"]').type('Hostname{enter}'); // Filter -> 'Host name'
     cy.get('input[id="react-select-7-input"]').type('Service{enter}'); // Filter -> 'Service'
-
-    cy.get('input[id="input_Host"]').type('{enter}'); // Hostname -> <current host> (one entry only)
```

Comment:
was this (and the other deleted lines) deleted by accident? most of them look relevant to me.